GatherPress/gatherpress
# Allow the date- and time-format strings to contain escaped characters

The current conversion from php dateformat into javascript moment format is not able to handle escaped characters.

### Description of the Change

My requested change makes the format conversion aware of escaped characters and tells it, to just go on, in such case.


Closes #632

### How to test the Change


1. Go to Events > Settings and set the date format as "j \d\e F \d\e Y"
2. Create an event
3. Make sure the shown date in the editor looks like "DAY de MONTH de YEAR"

### Changelog Entry

> Fixed - Bug fix Event date display in the editor when using escaped characters in date or time format


### Credits

Props @carstingaxion 

### Checklist:
